charts/argo-cd/templates/redis/service.yaml

```diff
@@ -11,7 +11,8 @@ metadata:
     {{- end }}
   {{- with .Values.redis.service.annotations }}
   annotations:
-    {{- toYaml . | nindent 4 }}
+  {{- range $key, $value := . }}
+    {{ $key }}: {{ $value | quote }}
```

Comment:
`{{- end }}` missing
```suggestion
    {{- range $key, $value := . }}
    {{ $key }}: {{ $value | quote }}
    {{- end }}
```

Reply:
Thanks for your patience. I'm new to this template language, and don't have a proper setup to test this here...

Reply:
No problem, we are happy to see contributions 🙏